### PR TITLE
Add more information about Go materials. Pdf optimization

### PR DESCRIPTION
**Pull request by samurai-of-honor**

### CODE REVIEW

**Comment by samurai-of-honor** [comment]
Which version is better? All on one line or starting from a new one?

**Comment by samurai-of-honor** [comment]
Which version is better? All in one column or on a new line?

File: Go/README.md [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
 + [Repl.it](https://repl.it/)
-+ [go.dev](https://go.dev/play/) (no data entry)
+
+Самый удобный сервис с <b>поддержкой терминала</b> и полноценного <b>ввода</b>.<br>
+Также есть возможность <b>совместной разработки</b> и поддержка множества <b>других языков.</b>
+
++ [go.dev](https://go.dev/play/)
+
+Браузерный редактор от <b>разработчиков Golang.</b><br>
+<b>Отсутствует</b> возможность ввода при работе програмы <b>но есть интересные примеры кода.</b>
```
Comment (SergeyOcheretenko) [inline Go/README.md:21]: Add , before "но" please

**Review by SergeyOcheretenko — CHANGES_REQUESTED** [review]

**Comment by SergeyOcheretenko** [comment]
Try to delete <br> tags and use markdown spacing

**Review by SergeyOcheretenko — APPROVED** [review]

**Pull request merged**